src/dashboards/hacs-dashboard.ts

```
@@ -179,37 +179,37 @@ export class HacsDashboard extends LitElement {
                 entry.divider
                   ? html`<li divider role="separator"></li>`
                   : html`
-                      <ha-menu-item
+                      <ha-md-menu-item
                         class="${entry.error ? "error" : entry.warning ? "warning" : ""}"
                         .clickAction=${() => {
                           entry?.action && entry.action();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)